src/main/java/de/hysky/skyblocker/skyblock/garden/VisitorHelper.java

```diff
@@ -107,6 +120,8 @@ private static void updateItemMap(String visitorName, String lore) {
     }
 
     private static void drawScreenItems(DrawContext context, TextRenderer textRenderer, int mouseX, int mouseY) {
+        context.getMatrices().push();
+        context.getMatrices().translate(0, 0, 232);
```

Comment:
Any particular reason as to why 232 is used?

Reply:
Same as cursor stack.

Reply:
The text for the count can still be covered by an item, perhaps try 400?